debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/gccheck/CheckEngine.java

```diff
@@ -108,6 +108,11 @@ public CheckEngine(J9JavaVMPointer vm, CheckReporter reporter)
 	{
 		_javaVM = vm;
 		_reporter = reporter;
+		try {
+			/* Even if hrm is null, all helpers that use it will null check it and attempt to allocate it and handle the failure */
+			MM_HeapRegionManagerPointer hrmPtr = MM_GCExtensionsPointer.cast(_javaVM.gcExtensions()).heapRegionManager();
+			_hrm = GCHeapRegionManager.fromHeapRegionManager(hrmPtr);
+		} catch (CorruptDataException cde) {}
```

Comment:
This exception ought not to be ignored: I expect many NullPointerExceptions or similar problems if `_hrm` is null.